### PR TITLE
Add KR 2023

### PR DESCRIPTION
**Pull request by Whisht**
### Which conference does this PR update?
- KR 2023

### Related URL

- https://kr.org/KR2023/
- https://dblp.uni-trier.de/db/conf/kr/index.html

### CODE REVIEW

**Review by RyunMi — APPROVED** [review]

**Review by RyunMi — CHANGES_REQUESTED** [review]
abstract_deadline -> deadline

**Comment by xuhangc** [comment]
time format incorrect

File: conference/AI/aamas.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  dblp: atal
+  confs:
+    - year: 2023
+      id: aamas23
+      link: https://aamas2023.soton.ac.uk/
+      timeline:
+        - abstract_deadline: '2022-10-21 00:00:00'
+          comment: 'abstract_deadline'
+        - deadline: '2022-10-28 00:00:00'
+          comment: 'paper submission deadline'
```
Comment (cubercsl) [inline conference/AI/aamas.yml:14]:
```suggestion
        - abstract_deadline: '2022-10-21 00:00:00'
          deadline: '2022-10-28 00:00:00'
```

File: conference/AI/kr.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  dblp: kr
+  confs:
+    - year: 2023
+      id: krd23
+      link: https://kr.org/KR2023/
+      timeline:
+        - abstract_deadline: '2023-03-03 00:00:00'
+          comment: 'submission of title and abstract'
+        - deadline: '2023-03-14 00:00:00'
+          comment: 'paper submission'
```
Comment (cubercsl) [inline conference/AI/kr.yml:14]:
```suggestion
        - abstract_deadline: '2023-03-03 00:00:00'
          deadline: '2023-03-14 00:00:00'
```

**Review by cubercsl — CHANGES_REQUESTED** [review]

**Pull request merged**